public/app/features/dashboard/components/PubdashFooter/PubdashFooter.tsx

```diff
@@ -0,0 +1,39 @@
+import { css } from '@emotion/css';
+import React from 'react';
+
+import { GrafanaTheme2, colorManipulator } from '@grafana/data';
+import { useStyles2 } from '@grafana/ui';
+
+export default function PubdashFooter() {
+  const styles = useStyles2(getStyles);
+
+  return (
+    <div className={styles.footer}>
+      <span className={styles.logoText}>
+        <a href="https://grafana.com/" target="blank">
+          powered by Grafana <img className={styles.logoImg} src="public/img/grafana_icon.svg"></img>
+        </a>
+      </span>
+    </div>
+  );
+}
+
+const getStyles = (theme: GrafanaTheme2) => ({
+  footer: css`
+    position: absolute;
```

Comment:
Do we need this footer to be absolutely positioned, or can we instead add extra bottom padding to the dashboard and display it there instead?

Reply:
In this case, I thought it would be better to absolutely position it and just do a transparent background in order to take up less screen real estate. I will play around with your suggestion and see how it looks though.

Reply:
Using padding makes this a more intrusive change, and considering that this only rendered in one case, I'm not sure it's worth it here. I'm inclined to just leave a todo to assess if we want this displayed in other instances and maybe refactor if so.